03-Custom-Actions-In-Python/03-Custom-Action-Features.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>I think it is confusing to declare that "..action will have its <code>act</code> method called &nbsp;it will have an attribute <code>_state</code> accessible..." and the also demonstrate a way to break this.</p><p> </p><p>If you expect users to have to modify <code>attach</code>,  I would use a positive example. For example, demonstrate saving something other than <code>_state</code>  and show that the user should call <code>super().attach(simulation)</code>  because it is responsible for storing <code>_state</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>